skimage/segmentation/active_contour_model.py

```diff
@@ -97,24 +94,10 @@ def active_contour(image, snake, alpha=0.01, beta=0.1,
     25
 
     """
-    if bc is not None:
-        message = ('The keyword argument `bc` to `active_contour` has been '
-                   'renamed. Use `boundary_condition=` instead. `bc` will be '
-                   'removed in scikit-image v0.18.')
-        warn(message, stacklevel=2)
-        boundary_condition = bc
-    if coordinates is None:
-        message = ('The coordinates used by `active_contour` will change '
-                   'from xy coordinates (transposed from image dimensions) to '
-                   'rc coordinates in scikit-image 0.18. Set '
-                   "`coordinates='rc'` to silence this warning. "
-                   "`coordinates='xy'` will restore the old behavior until "
-                   '0.18, but will stop working thereafter.')
-        warn(message, category=FutureWarning, stacklevel=2)
-        coordinates = 'xy'
-        snake_xy = snake
-    if coordinates == 'rc':
-        snake_xy = snake[:, ::-1]
+    if coordinates != 'rc':
+        raise ValueError('Coordinate values must be set in a row column '
+                         'format. `coordinates` must be indifferently set '
+                         'to "rc" or None.')
```

Comment:
Well, this message is now incorrect: `coordinates=None` will raise this error!

Reply:
Sorry, I changed, rechanged, re-rechanged and forgot to re*3 change.